services/src/test/resources/describe/wdl_draft_2/invalid_workflow/description.json

```diff
@@ -12,7 +12,8 @@
   "images" : [
   ],
   "submittedDescriptorType" : {
-
+    "descriptorType" : "WDL",
+    "descriptorTypeVersion" : "draft-2"
```

Comment:
The workflow in this case does not in any way resemble a WDL, so it is quite surprising to me that we say it is.

Reply:
Like I see that we claim it's a WDL draft-2 in the `workflowType` and workflowDescription`, so is this PR more or less intending to echo back what was submitted there?

Reply:
Yes, this was my understanding of the ticket. @cjllanwarne, since you created this ticket, could you please clarify that?